sorc/ncep_post.fd/IFI.F

```diff
@@ -199,6 +199,7 @@ subroutine make_communicators
     ! and for determining root ranks.
     allocate(ista_grid(size))
     allocate(jsta_grid(size))
+
```

Comment:
```suggestion
```
Remove accidental blank line. This came from the inline post bug fix. It was removed from that PR, but hasn't been removed from this one yet.